drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -0,0 +1,327 @@
+          },
+        },
+        days_since_last_service_es_180_issues: {
+          title: 'Possible Missed Exit - ES, Time in Enrollment 180 Days or More',
+          description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have been in shelter more than 180 days',
+          limiter: ->(item) {
+            item.lot > 180 && GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:es].include?(item.project_type)
+          },
+        },
+        days_since_last_service_es_365_issues: {
+          title: 'Possible Missed Exit - ES, Time in Enrollment 365 Days or More',
+          description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have been in shelter more than 365 days',
+          limiter: ->(item) {
+            item.lot > 365 && GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:es].include?(item.project_type)
+          },
+        },
+        days_since_last_service_so_90_issues: {
+          title: 'Possible Missed Exit - SO, Time in Enrollment 90 Days or More',
+          description: 'There is an expectation that clients will not stay indefinitely in street outreach, these clients have been in street outreach with no current living situation collected for more than 90 days',
+          limiter: ->(item) {
+            item.days_since_last_service > 90 && GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:so].include?(item.project_type)
+          },
+        },
+        days_since_last_service_so_180_issues: {
+          title: 'Possible Missed Exit - SO, Time in Enrollment 180 Days or More',
+          description: 'There is an expectation that clients will not stay indefinitely in street outreach, these clients have been in street outreach with no current living situation collected for more than 180 days',
+          limiter: ->(item) {
+            item.days_since_last_service > 180 && GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:so].include?(item.project_type)
+          },
+        },
+        days_since_last_service_so_365_issues: {
+          title: 'Possible Missed Exit - SO, Time in Enrollment 365 Days or More',
+          description: 'There is an expectation that clients will not stay indefinitely in street outreach, these clients have been in street outreach with no current living situation collected for more than 365 days',
+          limiter: ->(item) {
+            item.days_since_last_service > 365 && GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES[:so].include?(item.project_type)
+          },
+        },
+        days_in_ph_prior_to_move_in_90_issues: {
+          title: 'Possible Missed Move In Date - PH, Time in Enrollment 90 Days or More',
+          description: 'There is an expectation that clients in PH will eventually move into houseing, these clients have been in PH without a move-in date or with an invalid move-in date more than 90 days',
```

Comment:
I expect this is from the tool, but:
```suggestion
          description: 'There is an expectation that clients in PH will eventually move into housing, these clients have been in PH without a move-in date more than 90 days,  or have an invalid move-in date ',
```
Same edit would apply for 180 and 365 days.

Reply:
Good catches.  Thanks!